flink-tests/src/test/java/org/apache/flink/test/classloading/ClassLoaderITCase.java

```diff
@@ -300,7 +300,8 @@ public void testCheckpointingCustomKvStateJobWithCustomClassLoader() throws IOEx
 	 */
 	@Test
 	public void testDisposeSavepointWithCustomKvState() throws Exception {
-		ClusterClient<?> clusterClient = new MiniClusterClient(new Configuration(), miniClusterResource.getMiniCluster());
+		Configuration configuration = new Configuration();
```

Comment:
nit: I guess it was extracted to disable unaligned checkpoints, but then CLI argument was used; so this variable can be inlined back.